### DISCUSSION TITLE
Small Setup UX Improvement: Accept address without leading http/s

### DISCUSSION THREAD

**Discussion by amo13** [astubenbord/paperless-mobile#377] [Feature Requests]
I wanted to onboard two persons on my paperless and they both failed at this step. (I know, even though the placeholder in the address field starts with "https://")

If the app could figure out by itself whether to use https (if available) or http  whenever no schema is entered in the address field on setup, that would be a small but awesome improvement!
I imagine the app could
- try https (prioritized) and if it not available try http
- assume https and ask for user confirmation if it should use a plan http connection, or
- use any other workflow?

That would reduce friction during the setup process. Not everyone understands the role http/s in a URI.

**Comment by astubenbord** [comment]
Hm, good suggestion, maybe a simple dropdown which defaults to https should do the trick?

**Reply by amo13** [reply]
Whichever way to feel like implementing =)

Personally, I'd hide it from the user unless necessary (e.g. because https as default did not work). 
You could make the app assume https and automatically try http if 443 is not responding. A user-supplied address with "http://xxxxxx:1234" would override the defaults for the schema and/or the port. And something like "Do you want to accept the self-signed certificate?" could be handled with a popup.
